saw-core/src/SAWCore/Term/Pretty.hs

```diff
@@ -654,6 +699,7 @@ ppTerm opts = ppTermWithNames opts emptyDisplayNameEnv
 ppTermInCtx :: PPS.Opts -> [LocalName] -> Term -> PPS.Doc
 ppTermInCtx opts ctx trm =
   runPPM opts emptyDisplayNameEnv $
+  withVarNames (Set.toList (termVarNames trm)) $
   flip (Fold.foldl' (\m x -> snd <$> withBoundVarM x m)) ctx $
   ppTermWithMemoTable PrecTerm True trm
```

Comment:
Does `withVarNames` need to be called in the implementation of `ppTermInBinder` as well?

Alternatively, would it suffice to just call `withVarNames` once in the implementation of `ppTermWithMemoTable`? All of the current call sites to `withVarNames` occur within close proximity of `ppTermWithMemoTable`, after all.

Reply:
Fixed in c469a1cb6e4b3b669953ad8beab5db7bca35d417.